hammer/utils/lib_utils.py

```diff
@@ -21,7 +21,7 @@ def get_time_unit(source: str) -> Optional[str]:
         try:
             match = next(line for line in lines if "time_unit" in line)
             # attibute syntax: time_unit : <value><unit> ;
-            unit = re.split(" : | ; ", match)[1]
+            unit = re.split(" : | ; ", match)[1].replace('"', "").replace(";", "")
```

Comment:
Ah, indeed Liberty user guide specifies it as `time_unit : "1ps";`. The libs I tested on had no quotation marks.
Suggest doing this instead. Should cover all cases of quote/no quote, spaces/no spaces.
`re.split(":|;", match)[1].strip().strip('\"')`